docs/open-source/code-of-conduct.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Code of conduct
+description: TODO
+---
+
+# Code of Conduct
```

Comment:
The below is copied from Qiskit SDK repo

Reply:
Did you talk with the qiskit devs about this page? I'm not sure if we should copy-paste it in here or just use an external link?

Reply:
I didn't talk to them about whether it should be in the docs page vs a link to Qiskit SDK repo. @1ucian0 any thoughts?

Reply:
I'm fine with duplicating the page, i think it's important content, but we just need to make sure it doesn't go out of date if the qiskit repo version changes

Reply:
just chatted with luciano, neither of us feel strongly about wether it should be in docs or a link out, so lets just go with whatever is simpler to implement :D

Reply:
To clarify, I was envisioning updating Qiskit/qiskit to point to the page once it goes live with text like:

> This project follows the code of conduct at docs.quantum.ibm.com/open-source/code-of-conduct.